.circleci/config.yml

```diff
@@ -9,13 +9,14 @@ jobs:
       - checkout
       - node/with-cache:
           steps:
-            - run: npm install
+            - run: sudo npm i -g @nuel/uniformly && npm i --production
             - run: npm test
```

Comment:
If we're doing this to remove dev dependencies from the deployment bundle, I'd suggest we leave this as a plain `npm install` and then - in the deploy step do a `npm prune --production` prior to deploying.

I think this leaves us open to having to build problems if we add dev dependencies that are needed in the build/test process in the future and we wont want to keep adding exceptions to this list by installing them globally each time.